apsi_diploma/apsi_diploma/settings.py

```diff
@@ -24,9 +27,9 @@
 SECRET_KEY = "django-insecure-wv8nxxyjd)nvpc8^n#!ahui0nw+$kcbhi6!@bomr=g)t(2uoj6"
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
+DEBUG = False
```

Comment:
It should be dependent on environment (and test to True by default)

In case of development env this option hardcoded to False will be annoying

Reply:
@pktiuk you raised a very important issue. Currently the only time we push something to a real environment is where we merge develop to master. Then the stuff goes to Heroku. We do not have a "development instance" on Heroku. We could potentially have it, but setting it up will be a thing of the next sprint. However, in the current state the DEBUG should be false. Removing the secret key is also a good option by the way :) Will do that!

Reply:
By developmenent instance I mean my computer.

Reply:
I want to launch it on my machine without messing with data in cloud

Reply:
Does "messing with data in cloud" still relate to this DEBUG? Don't quite see how that could happen. If it relates to the database, please check out my comment in the #11 pull request :)

Reply:
Added this comment to wrong issue :)